### ISSUE TITLE
question about Theorem 1

### ISSUE THREAD

**Issue by Wangpeiyi9979** [YujiaBao/Distributional-Signatures#17]
Hi, thanks for your nice work, I have some question about Theorem1:

1、"σ as a mapping from the vocabulary set V to itself, and it is a bijection." is its domain and range are both V? the σ means we always change one word to a fixed word? 

2、the paper says: "the word σ(w) should be a discriminative feature for the new task. This property makes sense since word-importance for classification should depend only on the relative differences between classes". 

After perturbation, the word changes, but the class labels y remain the same, the outputs of the attention generator are the same, so why the changed word can be an import word for the origin y? 

3、the paper says: “We also note that the classification performance on the query set perturbations can be different”

is this mean the perturbations can affect the classification result for the query set? But isn't it anti-perturbation that we're trying to prove?

**Comment by YujiaBao** [comment]
1. Yes. Here you can consider sigma as a permutation.
2. for different class, the trigger (important) word may be different. The idea is that we don't want the classifier to be overfitting to those important words that we have already seen during training and ignoring potential important words for the testing classes.
3. Here we want to point out that when combined with pre-trained representations, you may not have this guarantee (as now we are viewing words as continuous embeddings rather than one-hot representation). But still, we can empirically show the benefit of learning on top of the distributional signatures.

**Comment by Wangpeiyi9979** [comment]
thx
